src/components/ha-wa-dialog.ts

```diff
@@ -275,8 +275,10 @@ export class HaWaDialog extends LitElement {
 
           wa-dialog::part(dialog) {
             /* Make the dialog fill the whole screen height and not the safe height */
-            min-height: var(--ha-dialog-min-height, 100vh);
-            max-height: var(--ha-dialog-max-height, 100vh);
+            min-height: var(--ha-dialog-min-height, var(--safe-height));
+            max-height: var(--ha-dialog-max-height, var(--safe-height));
```

Comment:
```suggestion
            min-height: var(--ha-dialog-min-height, 100vh);
            max-height: var(--ha-dialog-max-height, 100vh);
```
We want it fullscreen. Padding are here to apply safe area

Reply:
That causes this on android 


![Screenshot_20251127-163628.png](https://github.com/user-attachments/assets/f2d4152b-58ce-4e48-b6f9-860679be95ae)

Reply:
With --safe-height:

![Screenshot_20251127-163727.png](https://github.com/user-attachments/assets/d1ad4920-6f30-42de-95b8-1344b1f5f99a)

Reply:
Try using `100dvh`. It's not because of safe area, it's because of browser header bar.

Reply:
ios 
`--safe-height`
<img width="585" height="1266" alt="Image" src="https://github.com/user-attachments/assets/4c868dc3-84ae-4d17-a30e-073b357b5896" />
`100vh` or `100dvh` (it's the same inside the app)
<img width="585" height="1266" alt="Image" src="https://github.com/user-attachments/assets/5e994f9f-f261-4a16-ac51-740ee89edc20" />

Reply:
```
min-height: var(--ha-dialog-min-height, 100vh);
min-height: var(--ha-dialog-min-height, 100dvh);
max-height: var(--ha-dialog-max-height, 100vh);
max-height: var(--ha-dialog-max-height, 100dvh);
```
This should fix the issue

Reply:
Hmm well that worked :shrug: 

I guess the x margins and dvh were the final fix here